src/main/java/cz/inventi/kpj/spring/springintroduction/config/GreetingConfigProperties.java

```diff
@@ -0,0 +1,12 @@
+package cz.inventi.kpj.spring.springintroduction.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
```

Comment:
Místo Component je lepsi dat nad nejakou configuraci napr. GreetingConfig... EnableConfigurationProperties... Ale uz to mas dobre, uznavam

Reply:
Dobre, zkusim se na to jeste kouknout. Ja tam dal Component kvuli Autowire v tamty tride. Diky:)

Reply:
To jsi udelal dobre... jen configuration properties se tam davaji pres EnableConfigurationProperties. V zasade pro tento kurz to je v pohode :)...
Ale v review tohle vzdycky vracim, takze jsem to aspon zminil :). Je to drobnost, uznal jsem to.

Reply:
Chapu diky. Ted jsem na to narazil v tech dokumentech, co jsi poslal a chapu:). Zkusil jsem to doladit a myslim, ze by to ted melo byt presne jak to ma byt:). A taky to uz chapu vic diky

Reply:
Přesně, ted už to je ideální 👍